lib/cdo/aws/cloud_formation.rb

```diff
@@ -216,15 +216,20 @@ def delete
       params.filter_map do |key, properties|
         value = CDO[key.underscore] || ENV.fetch(key.underscore.upcase, nil)
         param = {parameter_key: key}
+        # TEMPORARY: Logging where our stack parameters are coming from.
         if value
+          log.info "Using parameter #{key} from CDO (#{CDO[key.underscore]}) or Environment (#{ENV.fetch(key.underscore.upcase, nil)})"
```

Comment:
this logs the value, which is a security risk, changing this.

```suggestion
          log.info "Using parameter #{key} from #{CDO[key.underscore] ? 'CDO' : 'ENV'}"
```